src/gantt.ts

```diff
@@ -1726,6 +1726,7 @@ export class Gantt implements IVisual {
     private bindInteractivityService(tasks: Task[]): void {
         if (this.interactivityService) {
             const behaviorOptions: BehaviorOptions = {
+                rootElement: this.body,
                 clearCatcher: this.clearCatcher,
```

Comment:
code duplication, try instead
clearCather: this.body

Reply:
fixed in f9c7572